habitat/core/dataset.py

```diff
@@ -278,8 +278,10 @@ def __init__(
         cycle: bool = True,
         shuffle: bool = False,
         group_by_scene: bool = True,
-        max_scene_repeat: int = -1,
+        max_scene_repeat_episodes: int = -1,
+        max_scene_repeat_steps: int = -1,
         num_episode_sample: int = -1,
+        repetition_rand_interval: float = 0.2,
```

Comment:
Maybe add some explanation to `repetition_rand_interval ` in the docstring since one might not be able to tell what it does just by the variable name.  Also, is there a better name for it, since it's a bit weird to name a `float` type "interval".